src/rating/index.css

```diff
@@ -31,7 +31,8 @@
 
   cursor: pointer;
 
-  &.is-disabled {
+  &.is-disabled,
+  &.is-readonly {
```

Comment:
I think this is gonna have to be `is-readOnly` as it's two words, and this is more in line with `is-currentValue` and `is-outsideMonth` (used elsewhere).